react-form-controls/src/form-control.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+export default class Form extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      placeholder: true,
+      emailValue: ''
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      emailValue: event.target.value
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    console.log(this.state.emailValue);
+  }
+
+  render() {
+    return (
+      <form onSubmit={ this.handleSubmit }>
+        <label htmlFor="email">email
+          <input onChange={ this.handleChange } id="email" type="email" />
```

Comment:
Be sure to also pass a `value` prop to your input so that the `<input>` element is fully "controlled" by React.

```jsx
value={this.state.emailValue}
```